reviews.Rmd

```diff
@@ -3,23 +3,26 @@ title          : "Why does preregistration increase the persuasiveness of eviden
 authors        : "Aaron Peikert, Maximilian S. Ernst & Andreas M. Brandmaier"
 journal        : "Meta-Psychology"
 manuscript-id  : "4139"
-manuscript-src : "headings"
-
 cutout-quotes  : true
 class          : "draft"
 output         : papaja::revision_letter_pdf
+manuscript-src : manuscript.Rmd
 ---
 
 Dear Dr. Rickard Carlsson,
 
 thank you very much for taking the time to consider our manuscript for publication at _`r rmarkdown::metadata$journal`_.
 In the following we address your and each reviewers'/editors' concerns point-by-point.
-For Daniel Laken's review we took the liberty to additionally include a high level summary, since we found it, at times, hard to follow the different interwoven discussions.
 
 All changes of the manuscript since this review can be viewed on GitHub:
 
 [https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main](https://github.com/aaronpeikert/bayes-prereg/compare/v0.2.0...main#diff-a9a4aad3fa8c9c10c5404b632bc3a01a25d2d8430eb932bc35c76769963e4b70)
 
+For Daniel Laken's review we took the liberty to additionally include a high level summary, since we found it, at times, hard to follow the different interwoven discussions.
```

Comment:
make it less judgemental?

```suggestion
Concerning Daniel Laken's review, we took the liberty to provide a high level summary of his main points because they are repeated across different subdiscussions throughout his letter.
```